.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -7,7 +7,4 @@ Resolves: #issue-number-here
 - [ ] Added **tests** for changed code.
 - [ ] Updated **documentation** for changed code.
 
-<!-- **Note**: If your Pull Request introduces a new feature or changes the current behavior, it should be based
-on the `develop` branch. If it's a bug fix or only a documentation update, it should be based on the `master` branch. -->
```

Comment:
Are we not going to rebase against develop anymore unless its a bugfix or a documentation update?

Reply:
No more `develop` branch. So all changes go into master and any bug fixes gets backported to a release branch (eg: `1.0`) by a maintainer if critical/important. Otherwise, backports can also be contributed by community once the change is in master.

Reply:
Will updated contributor doc too at some point in the near future.